sdk/go/common/util/ciutil/vars_test.go

```diff
@@ -142,3 +142,18 @@ func TestDetectVarsDisableCIDetection(t *testing.T) {
 	v := DetectVars()
 	assert.Equal(t, "", v.BuildID)
 }
+
+//nolint:paralleltest // mutates environment variables
+func TestDetectVarsPrefersGenericCI(t *testing.T) {
+	// Arrange.
+	name := "generic-ci-system"
+
+	t.Setenv("PULUMI_CI_SYSTEM", name)
+	t.Setenv("TRAVIS", "true")
+
+	// Act.
+	v := DetectVars()
+
+	// Assert.
```

Comment:
I find these comments so noisy, and also not consistent with the other tests here.  Can we just remove them?

Reply:
Done :+1: